README.md

```diff
@@ -41,13 +41,17 @@ Node, or NPM. So we'll just assume you already have them all running well.
 
 Assuming you already have Node and NPM working, compilation is easy. Use the
 following shell code:
-
-    git clone https://github.com/pomf/pomf
-    cd pomf/
-    npm install
-    make
-
-After this, the pomf site is now compressed and set up inside `dist/`.
+```bash
+git clone https://github.com/pomf/pomf
+cd pomf/
+make
+make install
+```
+OR
```

Comment:
Well it is implied if you read literally one line down, but if you think it's worth being more explicit I'm happy to change it.